adam-core/src/test/scala/org/bdgenomics/adam/converters/AlignmentRecordConverterSuite.scala

```diff
@@ -222,7 +222,7 @@ class AlignmentRecordConverterSuite extends FunSuite {
 
   }
 
-  test("converting to fastq with unmapped reads") {
+  test("converting to fastq with unmapped reads when read reverse strand flag was set") {
```

Comment:
could you create a new unit test instead of modifying this existing one?  it is better to always be increasing unit test coverage